tests/fixtures/Search.khmer.json

```diff
@@ -0,0 +1,358 @@
+{
+  "languages": [
+      {
+          "id": "km",
+          "name": "Khmer (Battambang Khmer)",
+          "keyboards": [
+              "basic_kbdkhmr",
+              "basic_kbdkni",
+              "kbdkhmr",
+              "khmer10",
+              "khmer_angkor",
+              "sil_khmer"
+          ]
+      },
+      {
+          "id": "kxm",
+          "name": "Khmer, Northern (Khmer Lue)",
```

Comment:
Are these from the new alltags.json?

https://api.keyman.com/search?q=kxm currently returns
```
"name": "Khmer, Northern (Buriram)",
```

Reply:
Yes, they will be from langtags.json. The search is still under (re)construction.

Reply:
No. Sorry, they are not yet. My bad.

https://api.keyman.com/search?q=khmer gives the result you are expecting. The difference in language names returned  is one of the current issues in the keyboard search.